src/global/pid/pid.cc

```diff
@@ -7,8 +7,11 @@
 
```

Comment:
Moving everything to the same file clarifies data flow for LUT PID. Actual implementations for detectors can not be as uniform and probably not going to co-exist as well. That was my original reasoning.

Reply:
Is there ever going to be a situation where the PID from the LUT from some detectors are mixed with real PID from others?
I don't know if it's always possible but it certainly feels nicer not to jump back and forth between plugins.

Reply:
I'd say, yes, as soon as any PID reconstruction is production ready (works on full DIS events), we then don't keep its LUT around.

Reply:
What decade do you estimate that happening in? ;-)